crates/runtime/src/memory.rs

```diff
@@ -428,19 +429,24 @@ impl Memory {
                 make_accessible,
                 ..
             } => {
+                // Never exceed static memory size
                 if new_byte_size > base.len() {
                     return None;
```

Comment:
FWIW this is arguably a case that may want to get `memory_grow_failed` as well. This is akin to "malloc returned NULL" because the `base.len()` return value is the maximal size of the memory that the host is willing to allocate

Reply:
Agreed